app/views/layouts/application.html.erb

```diff
@@ -12,8 +12,15 @@
   </head>
 
   <body>
+    <%= render 'shared/navbar' %>
```

Comment:
nice job adding your navbar and cdn assets and flash messages to a partial! this helps clean up the code alot

Reply:
Thank you!